packages/contracts/contracts/core/SphinxModule.sol

```diff
@@ -162,9 +162,8 @@ contract SphinxModule is ReentrancyGuard, Enum, ISphinxModule, Initializable {
         if (numLeaves == 1) {
             emit SphinxMerkleRootCompleted(_root);
             state.status = MerkleRootStatus.COMPLETED;
-            // Set the active Merkle root to be `bytes32(0)` so that a new approval can occur in the
-            // future.
-            activeMerkleRoot = bytes32(0);
+            // We don't need to set the `activeMerkleRoot` to equal `bytes32(0)` because we never
+            // assign it to a non-zero value in this call.
```

Comment:
Nit: It's implied, but it would be nice to state explicitly this is b/c the only leaf is the `APPROVE` leaf.

Reply:
Updated this to be:

```
            // We don't need to set the `activeMerkleRoot` to equal `bytes32(0)` because it already
            // equals `bytes32(0)`. Since there's only an `APPROVE` leaf, the Merkle root was
            // approved and completed in this call.
```